### PR TITLE
RFC: Add JuliaCon accessibility doc

### PR DESCRIPTION
**Pull request by ranjanan**
Comments and suggestions welcome. 

cc: @aviks @xorJane

### CODE REVIEW

File: 2020/accessibility.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+<p>
+<b>
+Q: Can I bring a guide dog?
+</b></p>
+<p>
+A: Sure! Guide dogs and other service animals are welcome at JuliaCon 2020.
+</p>
+<p>
+<b>
+Q: Will speakers and audience members be using a microphone for asking questions?
```
Comment (xorJane) [inline 2020/accessibility.html]: Reword suggestion:

> Will speakers and audience members use a microphone to ask questions?

File: 2020/accessibility.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+</b></p>
+<p>
+A: The speakers will use microphones in all rooms. The audience members will be 
+using microphones in the keynote rooms, and in the smaller rooms we’ll ask the 
+speaker to repeat the question on the microphone.
```
Comment (xorJane) [inline 2020/accessibility.html]: Reword suggestion:

> Audience members will use microphones to ask questions in the keynote room, and in the smaller rooms, we'll ask speakers to repeat audience questions over the microphone.

File: 2020/accessibility.html [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+</p>
+<p>
+<b>
+Q: What food service will be provided?
+</b></p>
+<p>
+A: It is planned to have a meal service at the conference for all participants.
```
Comment (xorJane) [inline 2020/accessibility.html]: I think we should state what we're planning/where there's uncertainty. Last I recall, we were thinking we'd provide either breakfast or lunch. If that's still valid, maybe reword to something like

> We plan to provide one meal per day at the conference for all participants.

File: 2020/accessibility.html [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+<p>
+A: Yes. If you have an allergy or a dietary preference, 
+please indicate this on the registration form. We will aim to accommodate all of them.
+</p>
+<p>
+<b>
+Q: Will there be loud music, strobe lighting, spotlights, or other strong sensory experiences?
+</b></p>
+<p>
+A: No.
```
Comment (xorJane) [inline 2020/accessibility.html]: Perhaps we should add something here about flash photography, a la 

> No. If you are sensitive to flash photography, please let us know so that we can direct other attendees to avoid using their flash features.

**Comment by ranjanan** [comment]
Reworking this because the conference has now shifted online

**Comment by ranjanan** [comment]
@aviks / @xorJane please merge if you think this is ok
